src/ralph/cli.py

```diff
@@ -37,6 +37,24 @@
 BACKENDS = (lambda: DATABASE_BACKENDS + STORAGE_BACKENDS)()
 
 
+class ComaSeparatedKeyValueParamType(click.ParamType):
+    """Coma separated key=value parameter type"""
```

Comment:
```suggestion
    """Comma separated key=value parameter type"""
```

Reply:
Good catch! I'll fix this everywhere I made the typo :sweat_smile: